python/core/qgsvectorlayer.sip

```diff
@@ -685,6 +685,11 @@ class QgsVectorLayer : QgsMapLayer, QgsExpressionContextGenerator
     virtual QString getStyleFromDatabase( const QString& styleId, QString &msgError /Out/ );
 
     /**
+     * Will delete the named style corresponding to style id provided from the database
+     */
+    virtual void deleteStyleFromDatabase( const QString& styleId, QString &msgError /Out/ );
```

Comment:
I think `/Out/` parameters make mostly sense if the return value is already used for something else. Could we use the return value instead here?